src/cascadia/TerminalSettingsModel/ActionArgs.idl

```diff
@@ -122,10 +122,22 @@ namespace Microsoft.Terminal.Settings.Model
         All = 0xffffffff,
     };
 
-    [default_interface] runtimeclass NewTerminalArgs {
+    interface INewContentArgs {
+        String Type { get; };
+        Boolean Equals(INewContentArgs other);
+        UInt64 Hash();
+        INewContentArgs Copy();
+        String GenerateName();
+    };
+
+    [default_interface] runtimeclass BaseContentArgs : INewContentArgs{

```

Comment:
See in this case, it actually makes an empty interface called `BaseContentArgs`. An empty one!

Reply:
Prefer:
```suggestion
    runtimeclass BaseContentArgs : [default]INewContentArgs {
```